src/python/WMCore/Storage/DeleteMgr.py

```diff
@@ -253,68 +260,11 @@ def deletePFN(self, pfn, lfn, command):
         impl.retryPause = self.retryPauseTime
 
         try:
-            impl.removeFile(pfn)
+            if not self.bypassImpl:
+                impl.removeFile(pfn)
         except Exception as ex:
             self.logger.error("Failed to delete file: %s", pfn)
             ex.addInfo(Protocol=command, LFN=lfn, TargetPFN=pfn)
             raise ex
 
         return pfn
-
-    #    def reportStageOutFailure(self, stageOutExcep):
-    #        """
-    #        _reportStageOutFailure_
-    #
-    #        When a stage out failure occurs, report it to the input
-    #        framework job report.
-    #
-    #        - *stageOutExcep* : Instance of on of the StageOutError derived classes
-    #
-    #        """
-    #        errStatus = stageOutExcep.data["ErrorCode"]
-    #        errType = stageOutExcep.data["ErrorType"]
-    #        desc = stageOutExcep.message
-    #
-    #        errReport = self.inputReport.addError(errStatus, errType)
-    #        errReport['Description'] = desc
-    #        return
-
-    def searchTFC(self, lfn):
```

Comment:
And this is the 3rd implementation that I found for `searchTFC`. Has this one been replaced by searchRFC as well?

Reply:
Yes